### PR TITLE
Removed link. Closes #785

### LINKED ISSUE #785: scala tour 404's
http://docs.scala-lang.org/tutorials/tour/tour-of-scala.html

points to this 404

http://docs.scala-lang.org/tutorials/tour/automatic-closures.html

I've seen at least one other but can't remember where...

### PR DESCRIPTION
**Pull request by travissarles**

### CODE REVIEW

File: tutorials/tour/_posts/2017-02-13-tour-of-scala.md [hunk 1]
```diff
@@ -38,10 +38,7 @@ A [local type inference mechanism](local-type-inference.html) takes care that th
 
 ## Scala is extensible ##
 
-In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries:
-
-* any method may be used as an [infix or postfix operator](operators.html)
-* [closures are constructed automatically depending on the expected type](automatic-closures.html) (target typing).
```
Comment (Jasper-M) [inline tutorials/tour/_posts/2017-02-13-tour-of-scala.md]: I suspected they meant by-name arguments by this, and I appear to be right: http://www.scala-lang.org/old/node/138

File: tutorials/tour/_posts/2017-02-13-tour-of-scala.md [hunk 2]
```diff
@@ -38,10 +38,7 @@ A [local type inference mechanism](local-type-inference.html) takes care that th
...
+In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries. Any method may be used as an [infix or postfix operator](operators.html).
```
Comment (som-snytt) [inline tutorials/tour/_posts/2017-02-13-tour-of-scala.md]: ... except in context of a DSL.

File: tutorials/tour/_posts/2017-02-13-tour-of-scala.md [hunk 3]
```diff
@@ -38,10 +38,7 @@ A [local type inference mechanism](local-type-inference.html) takes care that th
...
 
 A joint use of both features facilitates the definition of new statements without extending the syntax and without using macro-like meta-programming facilities.
```
Comment (som-snytt) [inline tutorials/tour/_posts/2017-02-13-tour-of-scala.md]: "both" features as in Jasper-M's link to the classic text. Also, by "macro-like", today we mean "macros." Maybe, "without using metaprogramming facilities such as macros." (They don't hyphenate meta-programming on scalameta.org.)

File: tutorials/tour/_posts/2017-02-13-tour-of-scala.md (hunk 2 above)
Comment (Jasper-M) [inline tutorials/tour/_posts/2017-02-13-tour-of-scala.md]: Even in DSLs I thought people prefer to throw in the `.` these days. Although I don't see any mention in Dotty of doing anything about postfix ops, so maybe they are less dead than I thought... Anyway I personally prefer a dot before my op over a semicolon after it.

Comment (som-snytt) [inline tutorials/tour/_posts/2017-02-13-tour-of-scala.md]: I don't have my finger on the pulse of postfix usage.

Comment (SethTisue) [inline tutorials/tour/_posts/2017-02-13-tour-of-scala.md]: Agree, the mention of postfix should be dropped.

**Pull request merged**